internal/cli/apikeys/apikeys.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apikeys
```

Comment:
It might be better if we create a package `organization` under `apikeys`. Let me know what you think @gssbzn

Reply:
We have an organization package and given the hierarchy I'd say to put this package inside that package